src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -379,7 +379,7 @@
     through addr. We therefore need to disambiguate what kind of assignment
     we are doing based on something other than the refness of the left hand side. 
     -->
-    <Field Name="RefKind" Type="RefKind" Null="NotApplicable"/>
+    <Field Name="IsRef" Type="bool" Null="NotApplicable"/>
```

Comment:
ConditionalOperator uses "IsByRef".   Would it make sense to use the same in both cases? 
It does not matter which one, but "IsRef" sounds a bit better to me.